src/subscriptions/SubscriptionManager.h

```diff
@@ -20,18 +20,24 @@
 #pragma once
 
 #include <backend/BackendInterface.h>
-#include <config/Config.h>
-#include <log/Logger.h>
 #include <subscriptions/Message.h>
+#include <util/config/Config.h>
+#include <util/log/Logger.h>
 
 #include <memory>
 
+namespace clio {
+
+namespace web {
 class WsBase;
+}  // namespace web
+
+namespace subscription {
 
 class Subscription
```

Comment:
This class is a little bit confusing . it has subscribe function, as well as publish function. In other place, the instances of it are called subscribers. It not make sense seeing something "subscribers.publish(something)"

Reply:
Yea, i agree and we can rename it :+1: 
This pr however is mostly about namespaces - we better keep diffs as small as possible (and perform more refactorings in other dedicated prs). This diff is already pretty big as is..

Reply:
Let's leave the internals of these confusing classes for a later refactoring otherwise the diff will be even bigger than it already is.